README.md

```diff
@@ -124,29 +125,30 @@ Config.close_connection();
 You can also receive all available messages using a loop:
 
 ```typescript
-import { Config, Event, Listener } from "eventPeople";
-import { Base } from "eventPeople.listeners";
+import { Config, Event, Listener } from 'eventPeople';
+import { Base } from 'eventPeople.listeners';
 
 const event_name = 'payment.payments.pay.all';
 let has_events = true;
 
 while (has_events) {
-  has_events = false
-
-  await Listener.on("SOME_EVENT", (event: Event, context: Base) => {
-      has_events = true;
-      console.log("");
-      console.log(
-          `  - Received the "${event.name}" message from ${event.origin}:`
-      );
-      console.log(`     Message: ${event.body}`);
-      console.log("");
-      context.success();
-  });
+	has_events = false;
+
+	await Listener.on('SOME_EVENT', (event: Event, context: Base) => {
+		has_events = true;
```

Comment:
You need to insert .editorconfig file an use 2 spaces to indent code

Reply:
we already have a prettierrc file to handle this, make sure your vscode is properly configured to format code automatically